source/elements/oneDNN/source/primitives/index.rst

```diff
@@ -0,0 +1,91 @@
+..
+  Copyright 2019-2020 Intel Corporation
+
+.. default-domain:: cpp
+
+Primitives
+----------
+
+*Primitives* are functor objects that encapsulates a particular computation
+such as forward convolution, backward LSTM computations, or a data
+transformation operation. A single primitive can sometimes represent more
+complex fused computations such as a forward convolution followed by a ReLU.
+
+The most important difference between a primitive and a pure function is
+that a primitive can store state.
+
+One part of the primitive's state is immutable. For example, convolution
+primitives store parameters like tensor shapes and can pre-compute other
+dependent parameters like cache blocking. This approach allows DNNL primitives
+to pre-generate code specifically tailored for the operation to be performed.
+The DNNL programming model assumes that the time it takes to perform the
```

Comment:
2 times: DNNL --> oneDNN

(let me know if you intentionally preserved DNNL here)

Reply:
No, this is not intentional. I thought I fixed these :/